postgres-connector/build.gradle

```diff
@@ -15,6 +15,17 @@ dependencies {
 
 description = 'Postgres Connector'
 
+//needs it's own PR and commit history
+//compileJava {
+//    options.compilerArgs << '-Xlint:unchecked'
+//}
+
 java {
     withJavadocJar()
 }
+
+jar {
+  from {
+    configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
```

Comment:
Always a dilemma - won't work if we don't include, duplication if we do. At least in gradle it's easier to drop specific dependencies as we wish (like base egeria ones). However we're not at that point yet so I would completely agree with this proposal for now